coding-guides/making-your-own-starboard.md

```diff
@@ -156,7 +156,7 @@ module.exports = class {
     if (!stars) {
       const image = message.attachments.size > 0 ? await this.extension(reaction, message.attachments.array()[0].url) : '';
       if (image === '' && message.cleanContent.length < 1) return message.channel.send(`${user}, you cannot star an empty message.`);
-      const embed = new RichEmbed()
+      const embed = new MessageEmbed()
         .setColor(15844367)
         .setDescription(message.cleanContent)
         .setAuthor(message.author.tag, message.author.displayAvatarURL)
```

Comment:
Check this and others, I think v12 now has most avatar/icon things as methods now, like displayAvatarUrl()

Reply:
ok!